### PR TITLE
[Term Entry] Python sets .symmetric_difference_update() #4618

### LINKED ISSUE #4618: [Term Entry] Python sets .symmetric_difference_update()
### Reason/inspiration (optional)

We would like a new term entry on the '.symmetric_difference_update()' method for Sets in Python. The entry should go in a new file under `docs/content/python/concepts/sets/terms/symmetric-difference-update/symmetric-difference-update.md`

The entry should include the following:

- A description of the function
- A ## Syntax section that includes the syntax of the function and its details
- An ## Example section that includes a ```python ``` codeblock and ```shell ``` that shows the example output.
- A ## Codebyte Example that includes a ```codebyte/python ``` runnable codeblock

Please refer to the [concept entry template](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by Ewa-Krysiak**
### Description

-added the .symmetric_difference_update() method for python

### Issue Solved

Closes #4618 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/python/concepts/sets/terms/symmetric-difference-update/symmetric-difference-update.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+## Codebyte Example
+
+
+```codebyte/python
+sweet_drawer_1 = set(['lollipops', 'cheescake', 'gummy bears', 'chocolate', 'lemon bars'])
+sweet_drawer_2 = set(['waffles', 'brownie', 'gummy bears', 'jelly beans', 'chocolate'])
+
+sweet_drawer_1.symmetric_difference_update(sweet_drawer_2)
+print(sweet_drawer_1)
```
Comment (letsbuildanything) [inline content/python/concepts/sets/terms/symmetric-difference-update/symmetric-difference-update.md:67]:
```suggestion
sweet_drawer_1 = set(['lollipops', 'cheescake', 'gummy bears', 'chocolate', 'lemon bars'])
sweet_drawer_2 = set(['waffles', 'brownie', 'gummy bears', 'jelly beans', 'chocolate'])

sweet_drawer_1 ^= sweet_drawer_2

print(sweet_drawer_1)
```

**Review by letsbuildanything — CHANGES_REQUESTED** [review]
@Ewa-Krysiak Thank you for contributing to the Codecademy Docs!
I've requested some changes. Please take a moment to address them. If you have any concerns or reasons to reject specific suggestions, feel free to discuss them.

**Review by letsbuildanything — APPROVED** [review]
@Ewa-Krysiak Thank you for making the changes!
It looks good for the second round of review.

**Review by mamtawardhani — APPROVED** [review]
Thank you for your contribution to the Codecademy Docs @Ewa-Krysiak!

The entry is ready to be merged! 😄

**Pull request merged**